### PR TITLE
ref(gpu-context): Replace getMeta (proxy) with _meta object - [TET-144]

### PR DESCRIPTION
**Pull request by priscilawebdev**
**What this PR does:**

- Replace `getMeta` in the gpu context with `event._meta `object
- Tweaked the gpu context code
- Add tests

### CODE REVIEW

File: static/app/components/events/contexts/runtime/getRuntimeKnownData.tsx [hunk 1]
```diff
@@ -1,33 +1,41 @@
...
+  meta: NonNullable<Event['_meta']>['user'];
+};
+
+export function getRuntimeKnownData({data, meta}: Props): KeyValueListData {
   const knownData: KeyValueListData = [];
 
-  const dataKeys = runTimerKnownDataValues.filter(runTimerKnownDataValue =>
-    defined(data[runTimerKnownDataValue])
-  );
+  const dataKeys = runtimeKnownDataValues.filter(runTimerKnownDataValue => {
```
Comment (andriisoldatenko) [inline static/app/components/events/contexts/runtime/getRuntimeKnownData.tsx:16]:
```suggestion
  const dataTypes = runtimeKnownDataValues.filter(runTimerKnownDataValue => {
```
wdyt?

File: static/app/components/events/contexts/runtime/getRuntimeKnownData.tsx [hunk 2]
```diff
@@ -1,33 +1,41 @@
...
+    if (!defined(data[runTimerKnownDataValue])) {
+      if (meta[runTimerKnownDataValue]) {
+        return true;
+      }
+      return false;
+    }
+    return true;
+  });
+
+  for (const type of dataKeys) {
```
Comment (andriisoldatenko) [inline static/app/components/events/contexts/runtime/getRuntimeKnownData.tsx:26]:
```suggestion
  for (const dataType of dataTypes) {
```
or
```suggestion
  for (const type of dataTypes) {
```

File: static/app/components/events/contexts/runtime/getRuntimeKnownData.tsx (hunk 1 above)
Comment (priscilawebdev) [reply static/app/components/events/contexts/runtime/getRuntimeKnownData.tsx:16]: most of the code has dataKeys as a const... do you mind if I address this in a separate PR? Then I can update all the occurrences

Comment (andriisoldatenko) [reply static/app/components/events/contexts/runtime/getRuntimeKnownData.tsx:16]: yeah sounds good.

**Review by andriisoldatenko — APPROVED** [review]

**Pull request merged**